fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -1185,6 +1185,32 @@ public Expression visitEncryptKey(DorisParser.EncryptKeyContext ctx) {
         return new EncryptKeyRef(new StringLiteral(db), new StringLiteral(key));
     }
 
+    @Override
+    public Expression visitUseFunction(DorisParser.UseFunctionContext ctx) {
+        List<Expression> arguments;
+        if (ctx.name.getText().equalsIgnoreCase("CHAR")) {
+            String usingArg = ctx.usingArg == null ? "utf8" : ctx.usingArg.getText();
+            arguments = ImmutableList.<Expression>builder()
+                    .add(new StringLiteral(usingArg))
+                    .addAll(visit(ctx.arguments, Expression.class))
+                    .build();
+        } else if (ctx.name.getText().equalsIgnoreCase("CONVERT")) {
```

Comment:
if `convert` without `using`, u should return `cast(arg0 as arg1)` other wise u should return `convert_to(arg0, arg1)`